.envrc

```diff
@@ -124,13 +124,13 @@ make setup-git-config
 # we're enforcing that virtualenv be in .venv{,3}, since future tooling e.g.
 # venv-update will rely on this.
 venv_name=".venv"
-python_version="python2.7"
+python_version="python3.6"
 
-if [ "$SENTRY_PYTHON3" = "1" ]; then
-    venv_name=".venv3"
-    python_version="python3.6"
+if [ "$SENTRY_PYTHON2" = "1" ]; then
+    venv_name=".venv2"
+    python_version="python2.7"
 
-    warn "Running in EXPERIMENTAL Python 3 mode... Good luck"
+    warn "Running in (Soon to be dropped) Python 2 mode... Good luck"
```

Comment:
Let's change the message to `Running in *DEPRECATED* python2!`, no good luck needed here :)